subworkflows/local/fastq_align_bwamem2/main.nf

```diff
@@ -59,7 +59,7 @@ workflow ALIGNMENT {
         .groupTuple()
         .map { id, case_id, sample_name, sample_type ->
             def meta = [
-                id         : id,
+                id         : sample_name.unique()[0],
```

Comment:
use .first() instead? Not sure what the most elegant is

Reply:
Fixed